<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgUserAttrs</class>
 <widget class="QDialog" name="DlgUserAttrs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户属性</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DlgUserAttrs {
	background: rgb(255,255,255);
}
QPushButton {
	background:rgb(57,129,252);
	color: rgb(255,255,255);
	border: 1px solid rgb(57,129,252);
	border-radius: 2px;
	min-height: 28px;
	min-width: 76px;
}
QPushButton:hover, QPushButton:pressed {
	background: rgb(5,117,197);
}
QPushButton:disabled {
	background: rgb(204,204,204);
	border: 1px solid rgb(204,204,204);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="userAttrsTable"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>新增</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemove">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveAll">
       <property name="text">
        <string>清除列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEdit">
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
